designs/requirement/plugins/org.obeonetwork.dsl.requirement.design/schema/blacklistedRequirementDecorator.exsd

```diff
@@ -0,0 +1,114 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="org.obeonetwork.dsl.requirement.design" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appInfo>
+         <meta.schema plugin="org.obeonetwork.dsl.requirement.design" id="blacklistedRequirementDecorator" name="blacklistedRequirementDecorator"/>
+      </appInfo>
+      <documentation>
+         This extension point allows plugins to specify labels of Sirius nodes that should not be decorated with requirements.
+      </documentation>
```

Comment:
This is the succinct documentatation related to this extension point @vrichard12